src/imagemodule.c

```diff
@@ -127,8 +132,15 @@ static PyObject* image_stackmean(PyObject *self, PyObject *args){
     goto error;
   }
   
-  if(stackmean((data_t*)PyArray_DATA(input), (data_t*)PyArray_DATA(mout),
-               (long int*)PyArray_DATA(nout), ndims, dims, norm)){
+  // Ok now we don't touch Python Object ... Release the GIL
+  Py_BEGIN_ALLOW_THREADS
+  
+  int retval = stackmean((data_t*)PyArray_DATA(input), (data_t*)PyArray_DATA(mout),
```

Comment:
do you need to grab and save these pointers out side of the allow threads block?